crates/build/README.md

```diff
@@ -0,0 +1,23 @@
+# contract-build
+
+A crate for building [`ink!`](https://github.com/paritytech/ink) smart contracts. Used by 
+[`cargo-contract`](https://github.com/paritytech/cargo-contract).
+
+## Usage
```

Comment:
We should try make this a [proper README compile test](https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#include-items-only-when-collecting-doctests). 

Not sure if we're able to pull files in for the `manifest_path` though

Reply:
Great idea. Done!

I guess it doesn't matter that the manifest path does not exist, `execute` will just return an `Err` so the test will pass.

I would add `no_run` but not sure how to add multiple code block attributes, and `rust` is already there.